pyproject.toml

```diff
@@ -37,15 +37,15 @@ kedro = ">=0.18.8,<0.19.0"
 click = ">=8.0.4"
 kfp = ">=1.8.12,<2.0"
 tabulate = ">=0.8.7"
-semver = "~=2.10"
-pydantic = "~=1.9.0"
+semver = ">=2.10,<4.0.0"
+pydantic = ">=1.9.0,<2.0.0"
 google-auth = "<3"
 google-cloud-scheduler = ">=2.3.2"
 google-cloud-iam = "<3"
 gcsfs = ">=2022.1"
 fsspec = ">=2022.1"
 google-cloud-storage = "<3.0.0"
-grpcio = "~=1.44.0"
+grpcio = ">=1.53.0"
 grpcio-status = "~=1.44.0"
```

Comment:
1) grpcio-status = "~=1.44.0" needs update..
  I believe we should sync grpcio-status with grpcio ; same monorepo / version - https://github.com/grpc/grpc/tree/master/src/python
2) I don't think need to set min to 1.53.0 -- but that's a maintainer choice.. For greater compatibility, we could set as ..
    - grpcio >=1.4.0,<2.0.0
    - grpcio-status >=1.4.0,<2.0.0

Reply:
+1, let's change this @millsks   and we will merge it :)

Reply:
I will get these updated tomorrow.  Thanks for the update!